content/cli/orch_cli/plugins.md

```diff
@@ -280,6 +280,72 @@ Plugin `e90b1a09-6b56-4a92-b9cd-5fc4ef32ab74` is now owned by user `admin`.
 ...
 {{< /highlight >}}
 
+
+### blueprint-labels
+
+#### blueprint-labels list
+`cfy plug blueprint-labels list [OPTIONS] PLUGIN_ID`
```

Comment:
We should probably use the full form (cfy plugins blueprint-labels...) in docs.

Reply:
Oh, right